src/main/cpp/t03_points.cpp

```diff
@@ -22,11 +22,29 @@
 //2 3
 
 #include "t03_points.h"
-#include <iostream>
-
+#include <iostream>#include <iostream>
+#include<set>
+#include<map>
+#include<cmath>
+#include <string>
 
 using namespace std;
 
 int t03_points() {
-    
+    multiset <double> dist;
+map <double,string> points;
+int n, x, y;
+double d;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The scope of the variable 'd' can be reduced.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4562198)